src/Pim/Bundle/CatalogBundle/Resources/config/query_builders.yml

```diff
@@ -10,6 +10,8 @@ parameters:
     pim_catalog.doctrine.query.filter.object_code_resolver.class: Pim\Bundle\CatalogBundle\Doctrine\Common\Filter\ObjectCodeResolver
     pim_catalog.doctrine.query.filter.dummy.class:              Pim\Bundle\CatalogBundle\Doctrine\Common\Filter\DummyFilter
 
+    pim_catalog.elastic_search.query.filter.string.class: Pim\Bundle\CatalogBundle\ElasticSearch\Filter\StringFilter
```

Comment:
not sure we should mark them as "elastic"
as here, it's just the PQB, we don't really care about the implementation

or maybe it's the opposite
the filter services should be defined in an elastic file

you can let it for now, and we can change later, no pb